cpp/examples/string_transforms/README.md

```diff
@@ -0,0 +1,28 @@
+# libcudf C++ examples using string transforms
+
+This C++ example demonstrates using libcudf transform API to access and create
+strings columns.
+
+The example source code loads a csv file and produces a transformed column from the table using the values from the tables.
+
+Four examples are included:
```

Comment:
What happened to the other ones? I think it would be good to show an example just returning a fixed-width integer.

Reply:
I added it back. I was informed it wasn't that important for the release blog. But it's a valuable example anyway